Basic-Car-Maintenance/Basic_Car_Maintenance.entitlements

```diff
@@ -10,5 +10,9 @@
 	<true/>
 	<key>com.apple.security.files.user-selected.read-only</key>
 	<true/>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)com.mikaelacaron.Shared</string>
+	</array>
```

Comment:
same question here too

Reply:
Yep. The `.` is included with the app identifier prefix.